examples/Demo/Shared/Pages/Persona/Examples/PersonaDefault.razor

```diff
@@ -0,0 +1,7 @@
+﻿<FluentPersona Name="Kevin Sturgis"
+               Status="PresenceStatus.Available"
+               StatusSize="PresenceBadgeSize.Small"
+               StatusTitle="He is available"
+               Image="https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png"
```

Comment:
Could you use the `@DataSource.SamplePicture` image? To be independent of external resources (which can be removed later)